doc/source/whatsnew/v0.18.2.txt

```diff
@@ -318,3 +319,5 @@ Bug Fixes
 
 
 - Bug in ``Categorical.remove_unused_categories()`` changes ``.codes`` dtype to platform int (:issue:`13261`)
+- Bug in ``DataFrame.describe()`` where percentile identifiers in index were always rounded to at most 1 decimal place (:issue:`13104`)
+- Bug in ``DataFrame.describe()`` raises ``TypeError`` when a data frame has a column index of type ``object`` and the ouput contains only columns with numeric names (:issue:`13288`)
```

Comment:
This is the one when `_shallow_copy` changes the dtype of the index, as in
```python
pd.DataFrame({'A': list("BCDE"), 0: [1,2,3,4]}).describe()
```
I'm not sure how to word it meaningfully. Maybe this would be better:
> Bug in Dataframe,describe() may raise TypeError when data frame columns have names of different types.

Previously, I tried something like:
> Bug in Dataframe,describe() corrupts a columns index of the output, which may raise TypeError when displaying.

Will be grateful for suggestions.